From 84ff2437892a27dc2f99c71983dfe35f96ec7559 Mon Sep 17 00:00:00 2001
Message-ID: <84ff2437892a27dc2f99c71983dfe35f96ec7559.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 22:04:16 +0000
Subject: [PATCH v3 45/66] gnu: go-github-com-aws-aws-sdk-go-v2: Move to (gnu
 packages golang-web).

* gnu/packages/golang.scm (go-github-com-aws-aws-sdk-go-v2): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

*  gnu/packages/version-control.scm: Add (gnu packages golang-web) module.

Change-Id: I6e980749d9f9c1373adfc717b0195a1b8c32d6af
---
 gnu/packages/golang-web.scm      | 26 ++++++++++++++++++++++++++
 gnu/packages/golang.scm          | 25 -------------------------
 gnu/packages/version-control.scm |  1 +
 3 files changed, 27 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index ae9ffd3fd2..1542c409fe 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -104,6 +104,32 @@ (define-public go-github-com-aws-sdk
      "This is the official AWS SDK for the Go programming language.")
     (license license:asl2.0)))
 
+(define-public go-github-com-aws-aws-sdk-go-v2
+  (package
+    (name "go-github-com-aws-aws-sdk-go-v2")
+    (version "1.17.3")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/aws/aws-sdk-go-v2")
+             (commit "v1.17.3")))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1a07xab1cn96iff7zvp5a82fzhqwl0i4bhplkm2h1qbkxgldn6x0"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/aws/aws-sdk-go-v2"))
+    (propagated-inputs
+     (list go-github-com-aws-smithy-go
+           go-github-com-google-go-cmp-cmp
+           go-github-com-jmespath-go-jmespath))
+    (home-page "https://github.com/aws/aws-sdk-go-v2")
+    (synopsis "AWS SDK for Go v2")
+    (description
+     "Package sdk is the official AWS SDK v2 for the Go programming language.")
+    (license license:asl2.0)))
+
 (define-public go-github-com-aws-smithy-go
   (package
     (name "go-github-com-aws-smithy-go")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 0e821306f2..a02da0fb49 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -11300,31 +11300,6 @@ (define-public go-github-com-dvsekhvalnov-jose2go
     (home-page "https://github.com/dvsekhvalnov/jose2go")
     (license license:expat)))
 
-(define-public go-github-com-aws-aws-sdk-go-v2
-  (package
-    (name "go-github-com-aws-aws-sdk-go-v2")
-    (version "1.17.3")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/aws/aws-sdk-go-v2")
-                    (commit "v1.17.3")))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "1a07xab1cn96iff7zvp5a82fzhqwl0i4bhplkm2h1qbkxgldn6x0"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/aws/aws-sdk-go-v2"))
-    (propagated-inputs (list go-github-com-jmespath-go-jmespath
-                             go-github-com-google-go-cmp-cmp
-                             go-github-com-aws-smithy-go))
-    (home-page "https://github.com/aws/aws-sdk-go-v2")
-    (synopsis "AWS SDK for Go v2")
-    (description
-     "Package sdk is the official AWS SDK v2 for the Go programming language.")
-    (license license:asl2.0)))
-
 (define-public go-github-com-aws-aws-sdk-go-v2-config
   (package
     (inherit go-github-com-aws-aws-sdk-go-v2)
diff --git a/gnu/packages/version-control.scm b/gnu/packages/version-control.scm
index cf9a2f7a4c..ffd4feb73d 100644
--- a/gnu/packages/version-control.scm
+++ b/gnu/packages/version-control.scm
@@ -105,6 +105,7 @@ (define-module (gnu packages version-control)
   #:use-module (gnu packages gnupg)
   #:use-module (gnu packages golang)
   #:use-module (gnu packages golang-check)
+  #:use-module (gnu packages golang-web)
   #:use-module (gnu packages groff)
   #:use-module (gnu packages guile)
   #:use-module (gnu packages guile-xyz)
-- 
2.41.0

